stubs/z3.pyi

```diff
@@ -1,4 +1,4 @@
-#  pylint: disable=unused-argument,invalid-name,no-self-use
+#  pylint: disable=unused-argument,invalid-name,no-self-use,pointless-statement
```

Comment:
Why did you re-add a useless suppression? I removed that in c20d8bb. This is what I get with pylint 2.3.1:
```
************* Module z3
stubs/z3.pyi:1:0: I0021: Useless suppression of 'pointless-statement' (useless-suppression)
```

Reply:
This must be a bug in the pylint 2.2.2 I had installed. I upgraded to 2.4.3 and removed the useless supressions (same above).